source/write/bulk-write.txt

```diff
@@ -0,0 +1,287 @@
+.. _pymongo-bulk-write:
+
+=====================
+Bulk Write Operations
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: insert, update, replace, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use bulk write operations with
+{+driver-short+}. You can perform multiple write operations
+in a single database call by using bulk operations.
+
+Consider a scenario in which you want to insert a document into a collection,
+update multiple other documents, then delete a document. Performing
+these operations by using their individual methods makes a call to the database
+for each method. You can instead use bulk write operations to perform
+these operations in a single database call.
+
+Performing Bulk Operations
```

Comment:
This title seems a little vague and is basically the point of the entire page. What if we called it something like "Define the Write Operations"?